frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FormBuilder.tsx

```diff
@@ -8,12 +8,18 @@ import { getVisibleFieldIds } from '~features/logic/utils'
 import { useBgColor } from '~features/public-form/components/PublicFormWrapper'
 
 import { useCreatePageSidebar } from '../../common/CreatePageSidebarContext'
+import {
+  EndPageState,
+  setToEditingEndPageSelector,
+  stateSelector,
```

Comment:
nit: should we name this to something that's clearer, like `endPageStateSelector` or something

Reply:
[f36ddbc](https://github.com/opengovsg/FormSG/pull/5073/commits/f36ddbcdcef159d34b5ec1e512810f3b459713a0)